docs/source/examples/scaling.md

```diff
@@ -0,0 +1,161 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Noise Scaling Methods
+
+In this tutorial we will review the differences between noise scaling methods used in [Zero-Noise Extrapolation](https://mitiq.readthedocs.io/en/stable/guide/zne.html) (ZNE).
+As a reminder, ZNE works by running multiple versions of the desired circuit, each intended to scale the noise up from the base-level achieved by the hardware.
+Experimentally these experiments are often performed by pulse stretching, but as a quantum programmer, we typically do not have access to such low-level control.
+For this reason we use "digital" methods that allow us to scale the noise using gate-based methods.
+To this end, we will study circuit folding and identity insertion as methods to increase the amount of noise present in our computation.
+
+As a reminder, these techniques are summarized by the following equations, and can be performed in Mitiq with the associated functions.
+
+|                | Folding              | Identity Insertion |
+| -------------- | -------------------- | ------------------ |
+| Equation       | $G \to GG^\dagger G$ | $G \to I G$        |
+| Mitiq Function | [`fold_global`](https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.zne.scaling.folding.fold_global) | [`insert_id_layers`](https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.zne.scaling.identity_insertion.insert_id_layers) |
+
+## Comparison
+
+To get started, we can demo what these two functions do to a small GHZ circuit.
+Each function (`fold_global` and `insert_id_layers`) take a circuit, and a specified scale factor which controls how much to increase the depth of the specified circuit.
```

Comment:
```suggestion
Each function (`fold_global` and `insert_id_layers`) will take a circuit, and a specified scale factor as inputs. The specified scale factor controls how to increase the depth of the input circuit so that the achieved scale factor is exactly equal or very close to the specified scale factor.
```